packages/ndla-ui/src/User/UserInfo.tsx

```diff
@@ -19,6 +19,10 @@ interface Props {
   user: FeideUserApiType;
 }
 
+const StyledComponentContainer = styled.div`
+  max-width: 700px;
```

Comment:
Kor kjem dette tallet fra? Korleis vil det sjå ut på mobil?

Reply:
Figma, denne komponenten skal ha max width 700px, og på mindre enheter så vil den bare fylle ut skjermen så lenge det er under 700px. Hvis dere ikke vil den her så kan jeg lage en ekstra container i frontend prosjeketet og fjerne det her.

Reply:
Neida, det treng du ikkje for min del. Eg er berre nysgjerrig.